android/beagle/src/main/java/br/com/zup/beagle/android/components/Image.kt

```diff
@@ -123,6 +127,13 @@ data class Image(
 }
 
 sealed class ImagePath {
-    data class Local(val mobileId: String) : ImagePath()
-    data class Remote(val url: String, val placeholder: Local? = null) : ImagePath()
+    data class Local(val mobileId: Bind<String>) : ImagePath() {
+        constructor(mobileId: String) : this(if (mobileId.isExpression()) expressionOf(mobileId) else valueOf(mobileId))
```

Comment:
You could extract this logic for an extension function as well as `valueOf` and `expressionOf`
```suggestion
        constructor(mobileId: String) : this(expressionOrValueOf(mobileId))
```

Reply:
I will create this function in another PR, to adjust other objects.